app/views/articles/_sidebar_campaign.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="widget">
+  <header>
+    <h4>
+      <% SiteConfig.campaign_featured_tags.each do |t| %>
+        <%= link_to("##{t}", "/t/#{t}") %>
+      <% end %>
+    </h4>
+  </header>
+  <div class="widget-body">
+    <div class="widget-link-list">
+      <% Article.tagged_with(SiteConfig.campaign_featured_tags, any: true).
```

Comment:
I know we do this in many places, but I'd slowly like us to move away from doing SQL queries from views. Is there another place where we could put this?